plugins/org.wso2.developerstudio.eclipse.gmf.esb.diagram/messages.properties

```diff
@@ -161,47 +161,47 @@ NavigatorGroupName_SpringMediatorOutputConnector_3062_outgoinglinks=outgoing lin
-FailoverEndPoint3CreationTool_desc=Create new Failover EndPoint
+FailoverEndPoint3CreationTool_desc=Defines the endpoints that the service will try to connect to in case of a failure. This will take place in a round robin manner
 NavigatorGroupName_FailoverEndPointInputConnector_3088_incominglinks=incoming links
 NavigatorGroupName_FailoverEndPointOutputConnector_3090_outgoinglinks=outgoing links
 WSDLEndPoint4CreationTool_title=WSDLEndPoint
-WSDLEndPoint4CreationTool_desc=Create new WSDL EndPoint
+WSDLEndPoint4CreationTool_desc=Defines the WSDL, Service and Port
 NavigatorGroupName_WSDLEndPointOutputConnector_3093_outgoinglinks=outgoing links
 NavigatorGroupName_WSDLEndPointInputConnector_3092_incominglinks=incoming links
 LoadBalanceEndPoint5CreationTool_title=Load-BalanceEndPoint
-LoadBalanceEndPoint5CreationTool_desc=Create new Load-Balance EndPoint
+LoadBalanceEndPoint5CreationTool_desc=Defines groups of endpoints for replicated services.The incoming requests will be directed to these endpoints in a round robin manner. These endpoints automatically handle the fail-over cases as well
 NavigatorGroupName_LoadBalanceEndPointOutputConnector_3096_outgoinglinks=outgoing links
 NavigatorGroupName_LoadBalanceEndPointInputConnector_3095_incominglinks=incoming links
 NavigatorGroupName_LoadBalanceEndPointWestOutputConnector_3098_outgoinglinks=outgoing links
 NavigatorGroupName_FailoverEndPointWestOutputConnector_3097_outgoinglinks=outgoing links
 HeaderMediator20CreationTool_title=Header
-HeaderMediator20CreationTool_desc=Create new Header Mediator
+HeaderMediator20CreationTool_desc=Sets or removes SOAP headers
 NavigatorGroupName_HeaderMediatorInputConnector_3100_incominglinks=incoming links
 NavigatorGroupName_HeaderMediatorOutputConnector_3101_outgoinglinks=outgoing links
 CloneMediator21CreationTool_title=Clone
-CloneMediator21CreationTool_desc=Create new Clone Mediator
+CloneMediator21CreationTool_desc=Clones a message
 IterateMediator22CreationTool_title=Iterate
-IterateMediator22CreationTool_desc=Create new Iterate Mediator
+IterateMediator22CreationTool_desc=Splits a message
 CacheMediator23CreationTool_title=Cache
-CacheMediator23CreationTool_desc=Create new Cache Mediator
+CacheMediator23CreationTool_desc=Evaluates messages based on whether the same message came to the ESB profile
 NavigatorGroupName_CloneMediatorInputConnector_3103_incominglinks=incoming links
 NavigatorGroupName_CloneMediatorOutputConnector_3104_outgoinglinks=outgoing links
 NavigatorGroupName_CacheMediatorOutputConnector_3107_outgoinglinks=outgoing links
 NavigatorGroupName_CacheMediatorInputConnector_3106_incominglinks=incoming links
 NavigatorGroupName_IterateMediatorOutputConnector_3110_outgoinglinks=outgoing links
 NavigatorGroupName_IterateMediatorInputConnector_3109_incominglinks=incoming links
 AggregateMediator24CreationTool_title=Aggregate
-AggregateMediator24CreationTool_desc=Create new Aggregate Mediator
+AggregateMediator24CreationTool_desc=Combines a message
```

Comment:
What if we say like combines the responses?

Reply:
Fixed.